### ISSUE TITLE
paddledetection迁移学习

### ISSUE THREAD

**Issue by gss-bai** [PaddlePaddle/PaddleDetection#1868]
同一批数据，先用fsater-rcnn进行模型训练，获得一个bestmodel，现在想用yolov3进行迁移学习，想请教下：
 yolov3配置文件yml中的pretrain_weights填写本地模型（bestmodel.pdparams）的文件路径吗？
谢谢！

**Comment by conan1024hao** [comment]
是的

**Comment by gss-bai** [comment]
> 是的

您好，我填写了本地路径进行训练，出现如下错误：

/home/shi/.local/lib/python3.6/site-packages/paddle/fluid/io.py:1998: UserWarning: This list is not set, Because of Paramerter not found in program. There are: conv1_1_weights bnv1_1_scale bnv1_1_offset conv1_2_weights bnv1_2_scale bnv1_2_offset conv1_3_weights bnv1_3_scale bnv1_3_offset res2a_branch2a_weights bn2a_branch2a_scale bn2a_branch2a_offset res2a_branch2b_weights bn2a_branch2b_scale bn2a_branch2b_offset res2a_branch2c_weights bn2a_branch2c_scale bn2a_branch2c_offset res2a_branch1_weights bn2a_branch1_scale bn2a_branch1_offset res2b_branch2a_weights bn2b_branch2a_scale bn2b_branch2a_offset res2b_branch2b_weights bn2b_branch2b_scale bn2b_branch2b_offset res2b_branch2c_weights bn2b_branch2c_scale bn2b_branch2c_offset res2c_branch2a_weights bn2c_branch2a_scale bn2c_branch2a_offset res2c_branch2b_weights bn2c_branch2b_scale bn2c_branch2b_offset res2c_branch2c_weights bn2c_branch2c_scale bn2c_branch2c_offset res3a_branch2a_weights bn3a_branch2a_scale bn3a_branch2a_offset res3a_branch2b_weights bn3a_branch2b_scale bn3a_branch2b_offset res3a_branch2c_weights bn3a_branch2c_scale bn3a_branch2c_offset res3a_branch1_weights bn3a_branch1_scale bn3a_branch1_offset res3b_branch2a_weights bn3b_branch2a_scale bn3b_branch2a_offset res3b_branch2b_weights bn3b_branch2b_scale bn3b_branch2b_offset res3b_branch2c_weights bn3b_branch2c_scale bn3b_branch2c_offset res3c_branch2a_weights bn3c_branch2a_scale bn3c_branch2a_offset res3c_branch2b_weights bn3c_branch2b_scale bn3c_branch2b_offset res3c_branch2c_weights bn3c_branch2c_scale bn3c_branch2c_offset res3d_branch2a_weights bn3d_branch2a_scale bn3d_branch2a_offset res3d_branch2b_weights bn3d_branch2b_scale bn3d_branch2b_offset res3d_branch2c_weights bn3d_branch2c_scale bn3d_branch2c_offset res4a_branch2a_weights bn4a_branch2a_scale bn4a_branch2a_offset res4a_branch2b_weights bn4a_branch2b_scale bn4a_branch2b_offset res4a_branch2c_weights bn4a_branch2c_scale bn4a_branch2c_offset res4a_branch1_weights bn4a_branch1_scale bn4a_branch1_offset res4b_branch2a_weights bn4b_branch2a_scale bn4b_branch2a_offset res4b_branch2b_weights bn4b_branch2b_scale bn4b_branch2b_offset res4b_branch2c_weights bn4b_branch2c_scale bn4b_branch2c_offset res4c_branch2a_weights bn4c_branch2a_scale bn4c_branch2a_offset res4c_branch2b_weights bn4c_branch2b_scale bn4c_branch2b_offset res4c_branch2c_weights bn4c_branch2c_scale bn4c_branch2c_offset res4d_branch2a_weights bn4d_branch2a_scale bn4d_branch2a_offset res4d_branch2b_weights bn4d_branch2b_scale bn4d_branch2b_offset res4d_branch2c_weights bn4d_branch2c_scale bn4d_branch2c_offset res4e_branch2a_weights bn4e_branch2a_scale bn4e_branch2a_offset res4e_branch2b_weights bn4e_branch2b_scale bn4e_branch2b_offset res4e_branch2c_weights bn4e_branch2c_scale bn4e_branch2c_offset res4f_branch2a_weights bn4f_branch2a_scale bn4f_branch2a_offset res4f_branch2b_weights bn4f_branch2b_scale bn4f_branch2b_offset res4f_branch2c_weights bn4f_branch2c_scale bn4f_branch2c_offset res5a_branch2a_weights bn5a_branch2a_scale bn5a_branch2a_offset res5a_branch2b_weights bn5a_branch2b_scale bn5a_branch2b_offset res5a_branch2c_weights bn5a_branch2c_scale bn5a_branch2c_offset res5a_branch1_weights bn5a_branch1_scale bn5a_branch1_offset res5b_branch2a_weights bn5b_branch2a_scale bn5b_branch2a_offset res5b_branch2b_weights bn5b_branch2b_scale bn5b_branch2b_offset res5b_branch2c_weights bn5b_branch2c_scale bn5b_branch2c_offset res5c_branch2a_weights bn5c_branch2a_scale bn5c_branch2a_offset res5c_branch2b_weights bn5c_branch2b_scale bn5c_branch2b_offset res5c_branch2c_weights bn5c_branch2c_scale bn5c_branch2c_offset fpn_inner_res5_sum_w fpn_inner_res5_sum_b fpn_inner_res4_sum_lateral_w fpn_inner_res4_sum_lateral_b fpn_inner_res3_sum_lateral_w fpn_inner_res3_sum_lateral_b fpn_inner_res2_sum_lateral_w fpn_inner_res2_sum_lateral_b fpn_res5_sum_w fpn_res5_sum_b fpn_res4_sum_w fpn_res4_sum_b fpn_res3_sum_w fpn_res3_sum_b fpn_res2_sum_w fpn_res2_sum_b conv_rpn_fpn2_w conv_rpn_fpn2_b rpn_cls_logits_fpn2_w rpn_cls_logits_fpn2_b rpn_bbox_pred_fpn2_w rpn_bbox_pred_fpn2_b fc6_w fc6_b fc7_w fc7_b cls_score_w cls_score_b bbox_pred_w bbox_pred_b
  format(" ".join(unused_para_list)))
2020-12-11 14:19:22,573-INFO: places would be ommited when DataLoader is not iterable
2020-12-11 14:19:23,829-INFO: iter: 0, lr: 0.000000, 'loss': '6436.328613', time: 0.000, eta: 0:00:02``

yml文件如下（只修改了pretrain_weights、num_classes、finetune_exclude_pretrained_params处）：
architecture: YOLOv3
use_gpu: true
max_iters: 70000
log_smooth_window: 20
save_dir: output
snapshot_iter: 2000
metric: VOC
map_type: 11point

pretrain_weights: output/qianyi_learn/model_final.pdparams
weights: output/yolov3_darknet_voc/model_final
num_classes: 4
use_fine_grained_loss: false

finetune_exclude_pretrained_params: ['yolo_output']

YOLOv3:
  backbone: DarkNet
  yolo_head: YOLOv3Head
